package.json

```diff
@@ -38,7 +38,7 @@
     "autocannon": "^7.0.5",
     "concurrently": "^6.1.0",
     "fastify": "^3.0.2",
-    "mercurius": "^8.0.0",
+    "mercurius": "github:mercurius-js/mercurius#master",
```

Comment:
the future Mercurius 8.10.0 is needed, since this commit is necessary:
https://github.com/mercurius-js/mercurius/commit/558929510b82e1ccbc1b8594ff36c78a5b07cd2c

Reply:
```suggestion
    "mercurius": "^8.10.0",
```